addons/web/static/src/js/views/kanban/kanban_view.js

```diff
@@ -242,9 +242,13 @@ var KanbanView = BasicView.extend({
     _updateMVCParams: function () {
         this._super.apply(this, arguments);
         var defaultGroupBy = this.arch.attrs.default_group_by;
-        this.loadParams.groupBy = defaultGroupBy ?
-                                    [defaultGroupBy] :
-                                    (this.loadParams.groupedBy || []);
+        if (this.searchMenuTypes.includes('groupBy')) {
```

Comment:
The parameter groupBy is only used as defaultGroupedBy in load. The query can still bring a search_default_groupby
(see param groupedBy).

Reply:
Maybe the change here is good but I would say it is a bad idea to put on the arch a default groupby if the view should not be 'groupable'.

Reply:
@Polymorphe57 Well, we handled it to avoid crash if someone set default  groupby using studio say for example in document kanban.

Reply:
@Polymorphe57 I can not do check in kanban_model.js as I don't have ```searchMenuTypes``` in kanban_model.js, for control_panel_model.js it is OK because we have ```searchMenuTypes``` is available in load method params there, but for kanban_model.js it is not the case, that's why I did this code in kanban_view.js, if you want then I can pass ```searchMenuTypes``` loadParams of kanban model and then we can put check there, what do you think?